lib/internal/cluster/round_robin_handle.js

```diff
@@ -13,7 +13,7 @@ function RoundRobinHandle(key, address, port, addressType, fd, flags) {
   this.free = [];
   this.handles = [];
   this.handle = null;
-  this.server = net.createServer(assert.fail);
+  this.server = net.createServer(() => { assert(false); });
```

Comment:
Can't we make this a top-level named function and reference it here for closure reuse?

Reply:
Ah yes, IIUC, it's now done, thanks.